scripts/blocks-utils.js

```diff
@@ -317,10 +317,17 @@ function parseResponse(apiResponse) {
   const result = [];
   apiResponse.Data.forEach((item) => {
     const jsonObject = {};
+    let excludeItem = false;
     item.forEach((data) => {
+      if (data.Key === 'TOTAL_COUNT') {
```

Comment:
@ravverma  can you please explain this one ?

Reply:
Current response from API , `Data` does have one json object which have `TOTAL_COUNT` only.